hooks/common/post-code-update/drush-env-switch.sh

```diff
@@ -38,6 +39,18 @@ drush @$drush_alias updatedb -y
 echo "Clearing caches"
 drush @$drush_alias cc all
 
+STOP_EXECUTION=false
```

Comment:
This step here needs to be removed since it would always override STOP_EXECUTION set during the custom step.